packages/expo-updates/src/Updates.types.ts

```diff
@@ -35,12 +64,16 @@ type UpdateCheckResultRollBackToEmbedded = {
    * Whether a roll back to embedded update is available.
    */
   isRollBackToEmbedded: true;
+  /**
+   * If no new update was found, this contains one of a set of enum values indicating the reason.
```

Comment:
```suggestion
   * If no new update is found, this contains one of several enum values indicating the reason.
```

Reading this definition it is not clear as to which "set" it is being implied here. Can we say "one of several enum values" here so that it indicates we have multiple enum values and any of it could be the `reason`.